src/app/common/ui-patternfly/syndesis-form-control.component.ts

```diff
@@ -48,20 +48,23 @@ export const enum SyndesisFormControlType {
 })
 export class SyndesisFormComponent extends DynamicFormControlComponent implements OnChanges {
 
+    @ContentChildren(DynamicTemplateDirective) contentTemplates: QueryList<DynamicTemplateDirective>;
+    // TODO disabling this for now as the base class is in a dependency
+    /* tslint:disable */
+    @Input('templates') inputTemplates: QueryList<DynamicTemplateDirective>;
+    /* tslint:enable */
+
     @Input() asBootstrapFormGroup= true;
     @Input() bindId= true;
     @Input() context: DynamicFormArrayGroupModel = null;
     @Input() group: FormGroup;
     @Input() hasErrorMessaging= false;
     @Input() model: DynamicFormControlModel;
-    @Input() nestedTemplates: QueryList<DynamicTemplateDirective>;
 
     @Output() blur: EventEmitter<DynamicFormControlEvent> = new EventEmitter<DynamicFormControlEvent>();
     @Output() change: EventEmitter<DynamicFormControlEvent> = new EventEmitter<DynamicFormControlEvent>();
     @Output() focus: EventEmitter<DynamicFormControlEvent> = new EventEmitter<DynamicFormControlEvent>();
 
```

Comment:
These input fields are assigned upon declaration hence do not need to be annotated with type tokens, since they will be typed by inference.